### ISSUE TITLE
GL-E750 Router with EG25-G Module 

### ISSUE THREAD

**Issue by bogdyn919** [srlabs/blue-merle#19]
Hi,  
Is possible to setup blue-merle  with GL-E750 Router with **EG25-G Module**  ?

**Comment by muelli** [comment]
Is the EG25-G built into the mudi v2?

Without having access to the hardware, it is difficult for us to determine how well blue-merle works. If you happen to have the hardware and can ship it to us, we can probably arrange for testing the device.

**Comment by richoffpints** [comment]
Was this ever proven to work?

**Comment by keepermanager** [comment]
Yes its work. I have the module EG25-G and work fine. Should be closed the issue.
